build.gradle

```diff
@@ -121,7 +121,7 @@ dependencies {
     compileOnly "com.github.spotbugs:spotbugs-annotations:$spotbugsAnnotationsVersion"
     compileOnly "com.google.code.findbugs:jsr305:$jsr305Version"
 
-    implementation "org.apache.kafka:kafka-clients:$kafkaVersion"
+    compileOnly "org.apache.kafka:kafka-clients:$kafkaVersion"
```

Comment:
Group to compile only, just one line up.

Reply:
I don't understand your message

Reply:
Move the line on up, having empty line between the compileOnly and implementation.

Reply:
done